README.md

```diff
@@ -113,6 +113,7 @@ Command Topics
 * [`box retention-policies`](docs/retention-policies.md) - List all retention policies for your enterprise
 * [`box search`](docs/search.md) - Search for files and folders in your Enterprise
 * [`box shared-links`](docs/shared-links.md) - Manage shared links
+* [`box sign-requests`](docs/sign-requests.md) - Manage sign requests
```

Comment:
This file is missing.

Reply:
I think these are automatically from the source code on release - take a look at commit history https://github.com/box/boxcli/tree/main/docs